staging/src/k8s.io/apiserver/pkg/storage/etcd3/watcher.go

```diff
@@ -261,8 +261,7 @@ func (wc *watchChan) processEvent(wg *sync.WaitGroup) {
 				continue
 			}
 			if len(wc.resultChan) == outgoingBufSize {
-				klog.V(3).Infof("Fast watcher, slow processing. Number of buffered events: %d."+
-					"Probably caused by slow dispatching events to watchers", outgoingBufSize)
+				klog.V(3).InfoS("Fast watcher, slow processing. Probably caused by slow dispatching events to watchers", "outgoingEvents", outgoingBufSize)
```

Comment:
This and the below message are actually important. I think we should make them V(2) or more, but wrap in a AtMost construct, since when they happen they could happen a lot. We added this to another such log message, I can search if you can't find it.